build-system/global-configs/client-side-experiments-config.json

```diff
@@ -23,6 +23,18 @@
     {
       "name": "story-load-inactive-outside-viewport",
       "percentage": 1
+    },
+    {
+      "name": "amp-sticky-ad-to-amp-ad-v2",
```

Comment:
Why is this added? Bad rebase?

Reply:
I'll look into it - yes, probably a bad rebase.